drivers/gpio/Kconfig

```diff
@@ -129,8 +129,12 @@ source "drivers/gpio/Kconfig.emul_sdl"
 
 source "drivers/gpio/Kconfig.psoc6"
 
+source "drivers/gpio/Kconfig.pcal64xxa"
+
 source "drivers/gpio/Kconfig.pcal6408a"
```

Comment:
Please only use `Kconfig.pcal64xxa` and remove `Kconfig.pcal6408a` and `Kconfig.pcal6416a`

Reply:
done